apps/balancer-tools/src/db/migrations/0000_striped_hiroim.sql

```diff
@@ -64,8 +64,6 @@ CREATE TABLE IF NOT EXISTS "pool_snapshots" (
 	"total_shares" numeric,
```

Comment:
generate new migrations instead of deleting this one, otherwise you'll force everyone to drop their databases